port/osx/omrintrospect.c

```diff
@@ -130,17 +139,29 @@ suspendAllPreemptive(PlatformWalkData *data)
 	}
 
 	if (0 == rc) {
+		sigset_t set = 0;
```

Comment:
This assumes that all `sigset_t` represents an empty set with all bits zero. The call to `sigemptyset()` below is the correct way to initialize it. Please remove the `= 0` initialization.